src/featureFlags/flags/taxServiceBanner.ts

```diff
@@ -0,0 +1,11 @@
+import { BaseVariant, FeatureFlag, useBaseFlag } from '../index'
+
+export function useTaxServiceBannerFlag(): BaseVariant {
+  return useBaseFlag(FeatureFlag.taxService, BaseVariant.Control)
```

Comment:
I generally try not to add wrapper hooks, since the `useBaseFlag(FeatureFlag.taxService, BaseVariant.Control)` line is super readable already and the wrapper adds no functionality